Update el.json

arius28 opened the pull request:
Translated the presets, plus 'clip'.

I didn't translate 'Sin City' since the effect is based on the looks of the movie and Sin City is how people mostly know it in Greece. Also left 'Jarques' as is. 

Should they be translated in Greek anyway? Is there a recommended approach on that? Jarques would just be written with Greek characters but Sin City is translatable.

arius28 committed:
Update el.json

Translated the presets, plus 'clip'.

I didn't translate 'Sin City' since the effect is based on the looks of the movie and Sin City is how people mostly know it in Greece. Also left 'Jarques' as is. 

Should they be translated in Greek anyway? Is there a recommended approach on that? Jarques would just be written with Greek characters but Sin City is translatable.

AnnaTomanek commented:
It's up to you to decide which version will be more understandable for Greek users. In case of the image editing tool the final effect on the photo is more important than its name, so it does not matter much, any solution that is acceptable for native users will be OK.

arius28 committed:
Update el.json

Changed some of the preset names to better reflect the underlying effect. Ready for merging!

AnnaTomanek commented:
Hi Aris, thanks again, your changes have just been merged and will be available in the next public release. Don't forget to [contact us](http://cksource.com/contact) about your free CKFinder license!

arius28 commented:
Hi Anna,

Thank you too!

On 29 May 2015 at 12:45, Anna Tomanek notifications@github.com wrote:

> Hi Aris, thanks again, your changes have just been merged and will be
> available in the next public release. Don't forget to contact us
> http://cksource.com/contact about your free CKFinder license!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ckfinder/ckfinder-translations/pull/13#issuecomment-106760088
> .
